src/core/cwa.ts

```diff
@@ -76,6 +82,41 @@ export default class Cwa {
     })
   }
 
+  async initMercure(headers) {
+    if (this.eventSource) return
+    const link = headers.link
+    if (!link) {
+      console.warn('No Link header found.')
+      return
+    }
+
+    const match = link.match(/<(.*)>.*rel="mercure".*/);
+    if (!match || !match[1]) {
+      console.log('No mercure rel in link header.')
+      return
+    }
+
+    const hub = new URL(match[1])
+    hub.searchParams.append('topic', '/_/routes/{id}')
+    hub.searchParams.append('topic', '/_/pages/{id}')
+    hub.searchParams.append('topic', '/_/layout/{id}')
+    hub.searchParams.append('topic', '/_/component_collections/{id}')
+    hub.searchParams.append('topic', '/component/html_contents/{id}')
```

Comment:
Should also listen for component position resource
`hub.searchParams.append('topic', '/_/component_positions/{id}')`

Reply:
Latest update subscribes to every known resource.